src/components/application_manager/test/commands/mobile/show_test.cc

```diff
@@ -765,6 +765,10 @@ TEST_F(ShowRequestTest, OnEvent_SuccessResultCode_SUCCESS) {
 
   SharedPtr<ShowRequest> command(CreateCommand<ShowRequest>(msg));
 
+  EXPECT_CALL(mock_message_helper_,
```

Comment:
@dev-gh Right but these changes are critical and needed for further development.